File: Backjoon1934/Backjoon1934.c
#include <stdio.h>
#include <stdlib.h>

int gcd(int num1, int num2){
    if(num2 == 0) return num1;
    else return gcd(num2, num1 % num2);
}

int main(){
    int T = 3;
    int a[3] = {1, 6, 13};
    int b[3] = {45000, 10, 17};
    int* A = (int*)malloc(sizeof(int) * T);
    int* B = (int*)malloc(sizeof(int) * T);
    for(int i = 0; i < T; i++){
        A[i] = a[i];
        B[i] = b[i];
    }
    for(int i = 0; i < T; i++)
        printf("%d\n", (A[i] * B[i]) / gcd(A[i], B[i]));
}

File: Backjoon1934/Backjoon1934.java
package Backjoon1934;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Backjoon1934 {
    public static int gcd(int num1, int num2){
        if(num2 == 0) return num1;
        else return gcd(num2, num1 % num2);
    }
    public static void main(String args[]) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;
        StringBuilder sb = new StringBuilder();
        int T = Integer.parseInt(br.readLine());
        for(int i = 0; i < T; i++){
            st = new StringTokenizer(br.readLine()); 
            int a = Integer.parseInt(st.nextToken());
            int b =Integer.parseInt(st.nextToken());
            sb.append((a * b) / gcd(a, b)).append("\n");
        }
        System.out.print(sb);
    }
}